packages/components/src/VirtualizedList/RowCollapsiblePanel/RowCollapsiblePanel.test.js

```diff
@@ -100,4 +102,28 @@ describe('RowCollapsiblePanel', () => {
 				.props.children({ measure: jest.fn() }),
 		).toMatchSnapshot();
 	});
+
+	it('should render a row with no data', () => {
+		// given
+		const noDataParent = {
+			...parent,
+			props: {
+				...parent.props,
+				rowGetter: () => ({}),
+			},
+		};
+
+		// when
+		const wrapper = mount(
+			<RowCollapsiblePanel
+				className={'my-class-names'}
+				index={1}
+				key={18}
+				parent={noDataParent}
+				style={{ background: 'red' }}
+			/>,
+		);
+
+		expect(toJsonWithoutI18n(wrapper)).toMatchSnapshot();
```

Comment:
```suggestion
		expect(toJsonWithoutI18n(wrapper.find('.tc-collapsible-row')).toMatchSnapshot();
```

Instead of snapshoting all the list, take a photo of the row instead, to ensure that it renders your loading component